Assets/Projectile/ProjectileWeapon.prefab

```diff
@@ -84,4 +84,5 @@ MonoBehaviour:
   TypeDefinition:
     WeaponPrefab: {fileID: 0}
     ProjectileType: 0
-    RateOfFire: 0.5
+    TimeBetweenShots: 0.3
```

Comment:
Did you actually change this for a specific reason or is it confused because of when I merged my changes?